test/commands/devtool.test.js

```diff
@@ -18,12 +18,16 @@ const assert = require("assert");
 const promisify = require('util').promisify;
 const sleep = promisify(setTimeout);
 const fetch = require('node-fetch');
-const DevToolCommand = require("../../src/commands/asset-compute/devtool");
+const mock = require('mock-require');
 
 const SERVER_START_UP_WAIT_TIME = 500; // ms to wait while server starts up
 const TIMEOUT = 3000;
 describe("devtool command", function() {
+    beforeEach( () => {
+        mock('open', () => {});
```

Comment:
I think you can just do that once at the top. Then you also don’t need the repeated DevToolCommand requires. Just make sure to mock open before requiring DevToolCommand.